cloud-template/src/main/java/com/sequenceiq/cloudbreak/cloud/template/network/NetworkResourceService.java

```diff
@@ -54,9 +54,9 @@
             AuthenticatedContext auth, Network network, Security security) throws Exception {
         CloudContext cloudContext = auth.getCloudContext();
         List<CloudResourceStatus> results = new ArrayList<>();
-        for (NetworkResourceBuilder builder : resourceBuilders.network(cloudContext.getPlatform())) {
+        for (NetworkResourceBuilder<ResourceBuilderContext> builder : resourceBuilders.network(cloudContext.getPlatform())) {
             PollGroup pollGroup = InMemoryStateStore.getStack(auth.getCloudContext().getId());
-            if (pollGroup != null && CANCELLED.equals(pollGroup)) {
+            if (CANCELLED.equals(pollGroup)) {
```

Comment:
is this cannot be null?

Reply:
Can be but unnecessary, equals also checks it